README.md

```diff
@@ -29,6 +29,12 @@ Or put the following in your Gemfile
 gem 'country_select'
 ```
 
+If you don't want to require `sort_alphabetical` (it depends on `unicode_utils` which is known to use lots of memory) you can opt out of using it as follows:
+
+```ruby
+gem 'country_select', require: 'country_select_without_unicode_utils'
```

Comment:
I know this is what I originally suggested, but I think I chose a bad filename. We can only guarantee that this new file won't `require 'sort_alphabetical'` so it should probably be named `country_select_without_sort_alphabetical.rb`

Sorry for the run around. What do you think?